docs/tutorials/working_with_the_ensemble.ipynb

```diff
@@ -19,7 +19,7 @@
   },
```

Comment:
<p>We have already discussed it above, and we have exactly the same <code>ens.source.compute()</code> line </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I'm also in favor of the repetition personally</p>

Reply:
Agree!